rrtmgp/kernels-openacc/mo_gas_optics_kernels.F90

```diff
@@ -561,8 +561,12 @@ subroutine compute_Planck_source(                        &
     real(wp), dimension(ngpt,     ncol), intent(out) :: sfc_src
     real(wp), dimension(ngpt,nlay,ncol), intent(out) :: lay_src
     real(wp), dimension(ngpt,nlay,ncol), intent(out) :: lev_src_inc, lev_src_dec
+
+    real(wp), dimension(ngpt,     ncol), intent(out) :: sfc_source_Jac
     ! -----------------
     ! local
+    real(wp), parameter                             :: dST = 1.0_wp
```

Comment:
Can this variable name be made more explicit, e.g. delta_Tsurf?

Reply:
Sure. Not a problem. Done

Reply:
If you have done this it's not yet visible - did you push your changes?